third_party/terraform/resources/resource_cloudfunctions_function.go

```diff
@@ -469,6 +506,11 @@ func resourceCloudFunctionsUpdate(d *schema.ResourceData, meta interface{}) erro
 		updateMaskArr = append(updateMaskArr, "timeout")
 	}
 
+	if d.HasChange("ingress_settings") {
+		function.IngressSettings = d.Get("ingress_settings").(string)
+		updateMaskArr = append(updateMaskArr, "ingress_settings")
```

Comment:
This will probably need to be `ingressSettings`- GCP APIs usually refer to fields using that form of case

Reply:
oh you are totally right sorry for that